### ISSUE TITLE
Dish links incorrectly labeled as omni

### ISSUE THREAD

**Issue by Starstrider42** [RemoteTechnologiesGroup/RemoteTech#67]
I have an intermittent bug where links are incorrectly drawn on the map view. In the attached image, all three sides of the triangular formation are dish-dish links (and should be displayed as such, not as omni links), and the two links to KSC are dish-omni. In fact, there are **no** active omni-omni links in this game.

![fakeomnilinks](https://cloud.githubusercontent.com/assets/6760782/3139649/95737976-e8e8-11e3-8d00-1eb36f81259f.png)

Persistence file and output_log are located at https://gist.github.com/Starstrider42/06462992a2ed15b06714. Tested with KSP 0.23.5, running RemoteTech 1.3.3 + Community Hotfix, Deadly Reentry 4.7, Kerbal Engineer Redux 0.6.2.4, Module Manager 2.1.4, and Toolbar 1.7.1.

To reproduce: 
- Load the save game linked in the Gist.
- Go to the tracking station and load Kerbal X (IMPORTANT: links are displayed correctly for any other mission in the save).
- Open map view

**Comment by KSP-TaxiService** [comment]
The persistence save is way too old, and DRE and KER are not updated for KSP 1.2 yet. I tested from scratch on KSP 1.2.1604 x64 and built out three copies of a probe with one Comm 16 (2.5 Mm), two Comm DTS-M1 (50.0 Mm) and some solar panels. I moved them to orbit via ALT+F12 window's orbit cheat.

![noproblem](https://cloud.githubusercontent.com/assets/22871614/20179849/f193c006-a792-11e6-8feb-c8d917e983d2.png)

The disk links in this pictures are colored properly (apart from the outdated Material strings disliked by Unity 5)
